.github/dependabot.yml

```diff
@@ -1,435 +1,807 @@
 # generated by dependadotnet
-# https://github.com/dotnet/core/tree/master/samples/dependadotnet
+# https://github.com/dotnet/core/tree/main/samples/dependadotnet
 version: 2
 updates:
-  - package-ecosystem: "github-actions" # Core GitHub Actions
```

Comment:
May want to manually add this one back, since the generator doesn't know about Actions yet